### ISSUE TITLE
Use of the from_data function

### ISSUE THREAD

**Issue by mbolt01** [dicompyler/dicompyler-core#10]
Hi,

_(I tried to post this on the google group, but it didn't seem to show up?)_

I am trying to access and use the `from_data()` function within dicompyler-core.

I am supplying my cumulative DVH data as a list and specifying the binsize = 1 (cGy) as 
`my_dvh =dvh.DVH.from_data(data=dvh_list,binsize=1)`

This is producing a DVH object with the following properties:
`DVH(cumulative, 100 bins: [0.0:100.0] Gy, volume: 5325 cm3, name: None, rx_dose: 0 Gy)`

Running` my_dvh.plot()` produced the following:
![dvh1](https://cloud.githubusercontent.com/assets/12694820/21848471/d09a2a52-d7f8-11e6-8858-130d515dd862.png)

but the cumulative DVH plot should be (based on plotting from the raw data):
![dvh2](https://cloud.githubusercontent.com/assets/12694820/21848474/d5e1846a-d7f8-11e6-9042-b23d425a8632.png)

So I was wondering if I am passing in the data incorrectly or misunderstanding the use of the `from_data` function. What format should the data that I pass to the function be in?
Also, is there a way to specify the prescription dose (`rx_dose`)?

Many thanks,

Matt

**Comment by bastula** [comment]
I'm linking the discussion on the dicompyler group here for reference:

https://groups.google.com/d/msg/dicompyler/E2GvESDqYaQ/GS70eVFSBwAJ

**Comment by bastula** [comment]
[Answer](
https://github.com/dicompyler/dicompyler-core/blob/master/dicompylercore/dvh.py#L21) from the discussion group:

`from_data` expects a raw dose array and will calculate a histogram and the number of bins (and bin size) for you based on this data passed in.

In your case, it would be better to instantiate the `DVH` class directly and pass
- the cumulative dose histogram counts as the `counts` argument
- the actual bins as `the bins` argument
- the rx dose as the `rx_dose` argument

i.e. `dvh.DVH(counts=dvh_list, bins=np.arange(0, len(dvh_list) + 1)`

See the DVH class initialization method for more information as all the parameters are documented:
[dvh.py](
https://github.com/dicompyler/dicompyler-core/blob/master/dicompylercore/dvh.py#L21)

I originally created the `from_data` class method when I first designed the `DVH` class, but quickly found that I didn't really use it after making the initializer more versatile. I figured someone might use this functionality when needing to calculate a DVH from raw dose data and did not remove it from the final class.
